--- /Users/jakubperdoch/Developer/UPR/Zadanie_6/test-spaces/stdout
+++ /Users/jakubperdoch/Developer/UPR/Zadanie_6/test-spaces/workdir/actual-stdout
@@ -18,22 +18,22 @@
 uppercase: 0 -> 2
 spaces: 0 -> 0
 
+
+lowercase: 0 -> 0
+uppercase: 0 -> 0
+spaces: 0 -> 0
+
 A
 lowercase: 46 -> 0
 uppercase: 0 -> 1
-spaces: 3 -> 0
-
-X
-lowercase: 1 -> 0
-uppercase: 0 -> 1
-spaces: 9 -> 0
-
-
-lowercase: 0 -> 0
-uppercase: 0 -> 0
 spaces: 3 -> 0
 
 
 lowercase: 0 -> 0
 uppercase: 0 -> 0
 spaces: 0 -> 0
+
+X
+lowercase: 1 -> 0
+uppercase: 0 -> 1
+spaces: 9 -> 0
