--- /Users/jakubperdoch/Developer/UPR/Zadanie_6/test-max-chars/stdout
+++ /Users/jakubperdoch/Developer/UPR/Zadanie_6/test-max-chars/workdir/actual-stdout
@@ -1,14 +1,14 @@
 ABCDABCDABCDABCDABCDABCDABCDABCDABCDABCDABCDABCDX
 lowercase: 49 -> 0
 uppercase: 0 -> 49
+spaces: 0 -> 0
+
+
+lowercase: 0 -> 0
+uppercase: 0 -> 0
 spaces: 0 -> 0
 
 ABCDABCDABCDABCDABCDABCDABCDABCDABCDABCDABCDABCDX
 lowercase: 49 -> 0
 uppercase: 0 -> 49
 spaces: 0 -> 0
-
-ABCDABCDABCDABCDABCDABCDABCDABCDABCDABCDABCDABCDX
-lowercase: 49 -> 0
-uppercase: 0 -> 49
-spaces: 0 -> 0
